README.md

```diff
@@ -1,3 +1,26 @@
 # Compatibility layer
 
 The compatibility layer of the EESSI project uses [Gentoo Prefix](https://wiki.gentoo.org/wiki/Project:Prefix)
+
+## Installation of Gentoo Prefix
+
+- Build a Singularity container with the provided definition file:
+```
+sudo singularity build prefix.simg singularity-prefix.def
+```
+
+- Run the bootstrap script inside the container:
+```
+singularity exec ./centos8.simg ./bootstrap-prefix.sh
```

Comment:
singularity exec should use consistent image name (prefix.simg or prefix.sif)

Reply:
Made the names consistent with the other command. Still using .simg here since I was running it with an older 2.x version on Ubuntu, but could use .sif (Singularity 3.x) as well.